torch/csrc/jit/python/init.cpp

```diff
@@ -1883,6 +1883,16 @@ void initJITBindings(PyObject* module) {
         return customObj->getPayload();
       },
       R"doc(Returns the Python object stored on the given opaque object.)doc");
+  m.def(
+      "_set_opaque_object_payload",
+      [](py::object obj, py::object payload) {
+        auto typePtr =
+            torch::getCustomClass("__torch__.torch.classes.aten.OpaqueObject");
+        auto ivalue = torch::jit::toIValue(std::move(obj), typePtr);
```

Comment:
what does the error message look like when you try to _set_opaque_object_payload on a scriptobject that is not a OpaqueObject? We want to check that that error message is reasonable

Reply:
It errors with
> RuntimeError: Tried to convert an IValue of type __torch__.torch.classes._TorchScriptTesting._TensorQueue (of Python compilation unit at: 0) to custom class type __torch__.torch.classes.aten.OpaqueObject (of Python compilation unit at: 0)

Reply:
nice